exporter/zipkinexporter/zipkin.go

```diff
@@ -39,10 +39,9 @@ import (
 
 // ZipkinConfig holds the configuration of a Zipkin exporter.
 type ZipkinConfig struct {
-	ServiceName      string         `mapstructure:"service_name,omitempty"`
-	Endpoint         string         `mapstructure:"endpoint,omitempty"`
-	LocalEndpointURI string         `mapstructure:"local_endpoint,omitempty"`
-	UploadPeriod     *time.Duration `mapstructure:"upload_period,omitempty"`
+	ServiceName  string         `mapstructure:"service_name,omitempty"`
+	Endpoint     string         `mapstructure:"endpoint,omitempty"`
+	UploadPeriod *time.Duration `mapstructure:"upload_period,omitempty"`
```

Comment:
Should this be added to new config?

Reply:
`UploadPeriod` is an option for the Zipkin exporter that i t should be handled in equivalent form via the pipeline, the same way that we do for OC (eg.: batcher, queued-retry).

`ServiceName` shouldn't actually be necessary, of course it still possible to have some bug in a client library not providing that information but it doesn't make much sense to inject it at the service as a configurable option, a place holder should suffice.